apps/console/src/features/applications/components/settings/access-configuration.tsx

```diff
@@ -635,116 +699,291 @@ export const AccessConfiguration: FunctionComponent<AccessConfigurationPropsInte
+                                                            isLoading={ isLoading }
+                                                            setIsLoading={ setIsLoading }
+                                                            certificate={ certificate }
+                                                            tenantDomain={ tenantName }
+                                                            allowedOrigins={ allowedOriginList }
+                                                            metadata={
+                                                                // There's no separate meta for `OAuth2/OIDC`
+                                                                // Apps. Need to use `OIDC` for noe.
+                                                                authProtocolMeta[
+                                                                    protocol ===
+                                                                                SupportedAuthProtocolTypes.OAUTH2_OIDC
+                                                                        ? SupportedAuthProtocolTypes.OIDC
+                                                                        : protocol
+                                                                ]
+                                                            }
+                                                            initialValues={
+                                                                get(
+                                                                    inboundProtocolConfig,
+                                                                    protocol ===
+                                                                                SupportedAuthProtocolTypes.OAUTH2_OIDC
+                                                                        ? SupportedAuthProtocolTypes.OIDC
+                                                                        : protocol
+                                                                )
+                                                                    ? inboundProtocolConfig[
+                                                                        protocol === SupportedAuthProtocolTypes
+                                                                            .OAUTH2_OIDC
+                                                                            ? SupportedAuthProtocolTypes.OIDC
+                                                                            : protocol
+                                                                    ]
+                                                                    : undefined
+                                                            }
+                                                            onSubmit={
+                                                                (values: any) => handleSubmit(values, protocol)
```

Comment:
AFAIU, we can use a more narrow type here, as follows.

```suggestion
                                                                (values: Record<string, FormValue>) => handleSubmit(values, protocol)
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5161/commits/f26e73598f340f7ac2bf1196034aedf5ac209429